libs/tokens/src/updaters/TokensListsUpdater/index.ts

```diff
@@ -80,12 +81,24 @@ export function TokensListsUpdater({ chainId: currentChainId }: TokensListsUpdat
 
     fetch('https://api.country.is')
       .then((res) => res.json())
-      .then(({ country }) => {
+      .then(({ country, ip }) => {
         const isUsUser = country === 'US'
 
         if (isUsUser) {
           setEnvironment({ useCuratedListOnly: true })
           updateLastUpdateTime({ [chainId]: 0 })
+
+          const sentryError = Object.assign(new Error(), {
+            message: '',
+            name: 'USUserError',
+          })
+
+          Sentry.captureException(sentryError, {
+            tags: {
+              errorType: 'USUserError',
+            },
+            contexts: { ip },
+          })
```

Comment:
Should this be in Sentry, though?
Feels more like analytics than an error.

Maybe we want to capture a message instead? https://docs.sentry.io/platforms/javascript/usage/#capturing-messages

Still, uncertain whether this is the right place for it.

Reply:
what i had in mind was more to know when this endpoint is not working, so to get a notification if the `catch` is called, not the `then`

Reply:
Fixed

Reply:
Looks great now! did you setup an alert in Sentry yet?